src/main/kotlin/org/rust/toml/TomlSchema.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml
+
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFileFactory
+import org.toml.lang.psi.TomlArrayTable
+import org.toml.lang.psi.TomlFileType
+import org.toml.lang.psi.TomlKeyValueOwner
+import org.toml.lang.psi.TomlTable
+
+class TomlSchema(project: Project, example: String) {
+    private val tables: List<TomlTableSchema>
+
+    init {
+        val toml = PsiFileFactory.getInstance(project)
```

Comment:
I prefer to move a complex logic from constructors to factory methods :)

Reply:
Will do!

Reply:
Fixed